api/src/stampy_chat/get_blocks.py

```diff
@@ -88,28 +90,38 @@ def get_top_k_blocks(index, user_query: str, k: int) -> List[Block]:
     blocks = []
     for match in query_response['matches']:
 
-        date = match['metadata']['date']
+        date = match['metadata']['date_published']
+
+        if isinstance(date, datetime.date):
+            date = date.isoformat()
+        elif isinstance(date, datetime.datetime):
+            date = date.date().isoformat
```

Comment:
Is this not meant to be `date = date.date().isoformat()`?

Reply:
good point